UnitTests/GitUI.Tests/ThreadHelperTests.cs

```diff
@@ -47,45 +46,6 @@ public async Task FileAndForgetIgnoresCancellationExceptions()
             Assert.Null(helper.Exception, helper.Message);
         }
 
-        [Test]
-        [Ignore("Hangs")]
```

Comment:
Any point in changing this to HandleExceptions()?
(If possible enabled)

Reply:
The removed testcases are duplicates of the remaining ones. They verified only that the removed argument `fileOnlyIf` worked as expected.